django/contrib/admin/options.py

```diff
@@ -869,7 +870,7 @@ def _get_base_actions(self):
         for (name, func) in self.admin_site.actions:
             if name in base_action_names:
                 continue
-            description = getattr(func, 'short_description', name.replace('_', ' '))
+            description = getattr(func, 'short_description', capfirst(name.replace('_', ' ')))
```

Comment:
Slightly unrelated/undiscussed change?

Reply:
That's why it's in a separate commit with the message `Refs #16117 -- Standardized admin action description generation.` The default name for the actions defined in `ModelAdmin.actions` uses `capfirst()`, but the default name for those defined in `AdminSite.actions` doesn't.

Sorry. There are a few cleanup commits in here where I found some warts while searching for all of the code related to these attributes.